build_helpers/startSiteDevServer.sh

```diff
@@ -7,4 +7,4 @@ rm -rf ./__site_prerender__
 ./build_helpers/buildAPIDocs.sh
 webpack --config "$PWD/site/webpack-prerender.config.js"
 ./build_helpers/buildSiteIndexPages.sh
-webpack-dev-server --config "$PWD/site/webpack-client.config.js" --no-info --content-base __site__
+webpack-dev-server --config "$PWD/site/webpack-client.config.js" --no-info --content-base __site__ --port ${1-8080}
```

Comment:
Unrelated to the fix, just a nice to have. Allows specifying an alternate port for the demo server, useful since LD uses 8080 as well.